### ISSUE TITLE
I'd like to add a Chinese doc translation, any structure suggested?

### ISSUE THREAD

**Issue by kxxoling** [avanov/Plim#42]
Should I just add a cn folder in the doc folder?

**Comment by avanov** [comment]
Hi @kxxoling 

Yes, that would be cool! Once the work is done, we'll add a link to the translation onto a main docs page.

Thanks!

**Comment by kxxoling** [comment]
@avanov Translation is under processing, but I found `syntax.txt` is a bit too large as a document. Would you like to split it to smaller parts first?

**Comment by avanov** [comment]
You can split it into multiple files and then include them with the ``.. include::`` directive into one document. I'd like to keep the syntax section as one document rather than a section with links to subsections.

**Comment by kxxoling** [comment]
@avanov So you won't split the English docs, but splited Chinese docs are acceptable?

**Comment by avanov** [comment]
Certainly. If I like your layout, I may probably adapt the English version later.

**Comment by kxxoling** [comment]
@avanov Almost finished but the syntax.txt! But I carelessly [added the changes to develop branch](https://github.com/kxxoling/Plim/tree/develop/docs/cn), need I move it to an independent branch?

**Comment by avanov** [comment]
@kxxoling develop is the right place to put your changes. When you do a pull request, please specify the develop branch of the upstream as the branch to merge to as well. Thanks!

**Comment by kxxoling** [comment]
@avanov Another question: where should I place the index.rst (cn version) and how should I reference the Chinese documents?

**Comment by kxxoling** [comment]
@avanov Any suggestion on the index.rst file?

**Comment by avanov** [comment]
Hey @kxxoling 
Sorry for a late response. Can you make a PR with the currently available changes. Then we'll think how to embed it onto the main docs page. Thanks!

**Comment by kxxoling** [comment]
@avanov I just learned a new way to build i18n language support, and want to change it into that way" http://sphinx-doc.org/latest/intl.html
But first I want to move the docs files in docs/en to docs directory which makes more rational.
Is that OK?

**Comment by avanov** [comment]
Is it supported by ReadTheDocs service?

**Comment by kxxoling** [comment]
@avanov Yes! RTFD supports it, but needs one more repository to display multi language at the same time: http://docs.readthedocs.org/en/latest/i18n.html

**Comment by avanov** [comment]
This link describes how i18n techniques are used in an underlying webapp that powers ReadTheDocs. What I'd like to know is whether ReadTheDocs service can build docs with sphinx-intnl automatically, as it does now with regular docs.

**Comment by avanov** [comment]
I see another section, though - http://docs.readthedocs.org/en/latest/localization.html

**Comment by avanov** [comment]
Ok, since it's still not completely clear to me how it works on ReadTheDocs, please do all the neccessary changes in your repo and then make a PR. Hopefully, it won't break current docs structure dramatically.

**Comment by kxxoling** [comment]
@avanov I don't think it would :) :beer:

**Comment by kxxoling** [comment]
Online Plim Chinese document is built successfully: http://klim.readthedocs.org/zh_CN/latest/ .
To add this as your Chinese version document, just add it in your ReadTheDocument dashboard: https://readthedocs.org/dashboard/plim/translations/

**Comment by avanov** [comment]
Can you rename it to plim-chinese for clarity?

**Comment by kxxoling** [comment]
Because RTFD seems not supporting rename project, I'm trying to delete it and re-add it. I'd like to rename it to plim-chs, which is short for Chinese Simplified, and won't be confused with Chinese Traditional, Chinese Wu, Chinese Yue.

**Comment by avanov** [comment]
Let me know when it's ready.

**Comment by kxxoling** [comment]
It's online now: http://plim-chs.readthedocs.org/zh_CN/ , but I need to build the .po files to .mo binaries as RTFD required, would you mind that to be merged into develop and master branch?

**Comment by avanov** [comment]
I don't mind.

**Comment by kxxoling** [comment]
Chinese version document is already online, you can now just marked it as a translation of your project

**Comment by avanov** [comment]
I've added ``plim-chs`` both as a translation and a subproject to the main Plim RTFD entry.

As the final step of this ticket I'd like to ask you to add yourself to the AUTHORS.

Thank you for this great contribution!

**Comment by kxxoling** [comment]
@avanov :) Thanks! I'd like to add myself in the next pull request.

**Comment by kxxoling** [comment]
I'm not sure but seems the .mo files are not necessary now maybe because ReadTheDocs have updated their processing. A example is here: http://fabric-chs.readthedocs.org/zh_CN/chs/ , which I haven't pushed the .mo files yet, but still works OK.
Will working on this later this week. It would be much cleaner to remove these .mo compiled files out of the repository.

**Comment by avanov** [comment]
No objections.
